sdk/keyvault/azure-keyvault-administration/azure/keyvault/administration/_access_control_client.py

```diff
@@ -27,18 +28,20 @@ class KeyVaultAccessControlClient(KeyVaultClientBase):
     # pylint:disable=protected-access
 
     @distributed_trace
-    def create_role_assignment(self, role_scope, role_assignment_name, role_definition_id, principal_id, **kwargs):
-        # type: (Union[str, KeyVaultRoleScope], Union[str, UUID], str, str, **Any) -> KeyVaultRoleAssignment
+    def create_role_assignment(
+        self, role_scope, role_definition_id, principal_id, **kwargs
```

Comment:
I think we landed on 120 columns for the SDK (`black -l 120`), and anyway I've used it everywhere, so many noisy PRs ahead if you want to start a revolution 😆

Reply:
Oh no, I don't mean to start a war! I can revert the formatting changes now that I know better